docs/recipes/Authentication.md

```diff
@@ -0,0 +1,1447 @@
+---
+title: Authentication with Supabase
+description: Here we use Supabase as our backend to show out to implement authentication with your React Native project.
+tags:
+  - authentication
+  - supabase
+  - login
+  - signup
+  - session
+last_update:
+  author: Nick Morgan (@morganick)
+  publish_date: 2024-04-12
+---
+
+## Overview
+
+Many applications require an external service to authenticate the user. Setting up authentication for your application can feel daunting. Where do I start? What data do I need from my users? What service(s) should or could I use? What are the signup, signin, and other user flows that I'll need?
+
+This recipe is going to use [Supabase](https://supabase.com) as the backend. We'll build some primatives that will allow you to customize the authentication to your needs or existing backend service as well.
+
+## Requirements
+
+Since we're using [Supabase](https://supabase.com) for our backend, it is assumed that you have an account there. We're going to need two pieces of information from that account the project URL and anonymous public key. (Inside your Supabase account, [visit the API credentials section](https://supabase.com/dashboard/project/_/settings/api).)
+
+## Starting Point
+
+We're going to start from a freshly ignited project without any of the boilerplate screens:
+
+```bash title="Terminal"
+bunx ignite-cli@latest new AuthRecipe --workflow=cng --remove-demo --git --install-deps
```

Comment:
just a thought, but most people don't use bun. maybe have alternatives for at least yarn